### ISSUE TITLE
Check that imaging data can be made public

### ISSUE THREAD

**Issue by ychae** [ncihtan/IDP#12]
From IDP 2/5 meeting: 

Ensure that imaging data in Harvard, OHSU and SARDANA Synapse projects live on buckets that can be made public (or at least imaging data can be made public)

**Comment by jaeddy** [comment]
Copying over my comment from Slack:

I’d be hesitant to make the source images fully public for various governance reasons — or at least, we'd need to review any potential risks before doing so. Also, if i understood correctly, Minerva just needs the _rendered_ images (along with waypoint/story files) to be public — basically the middle section of this image:

![image.png](https://images.zenhubusercontent.com/5df3dcf708c2dd496d1a7c1c/c64508d4-a716-4868-a1cc-1739c7ffb406)

**Comment by milen-sage** [comment]
Agreed w/ @jaeddy comment above; we need to prioritize Minerva artifacts being public and take care raw images are not made public prematurely.
